rental-car-manager/editorextensions/rental-car-manager/src/modal/rentacarmodal.ts

```diff
@@ -220,8 +226,31 @@ export class RentACarModal extends Modal {
     }
   }
 
+  private getOrCreateConditionalFormattingRules() {
+    const rules: RuleProxy[] = [];
+
+    const carColorRule =
+      this.documentProxy.rules.find(
+        (rule) => rule.getName() === CAR_COLOR_RULE_NAME,
+      ) || this.documentProxy.addRule(carColorRuleDefinition);
```

Comment:
extreme nit: the use of `||` feels a little opaque. Although maybe that's just me, maybe I haven't seen this pattern enough to be familiar with it

Reply:
It's basically the same thing as `??`, it only executes if the piece before it is falsy.